content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -77,10 +81,10 @@ The following is a comprehensive list of all known [amendments](amendments.html)
 ## CheckCashMakesTrustLine
 [CheckCashMakesTrustLine]: #checkcashmakestrustline
 
-| Amendment | CheckCashMakesTrustLine |
-|:----------|:-----------|
+| Amendment    | CheckCashMakesTrustLine |
```

Comment:
It's fine this time, but ideally, whitespace changes would be in a separate PR - so as to reduce noise when reviewing the salient changes.